README.md

```diff
@@ -1,2 +1,3 @@
 # hello-microverse
+
```

Comment:
- [ ] please It would be better if you use the [template](https://github.com/microverseinc/readme-template) that micoverse provides and customize it on your own

Reply:
This has been updated as instructed